### ISSUE TITLE
Infinity bits of entropy

### ISSUE THREAD

**Issue by nyuszika7h** [EFForg/privacybadgerfirefox#637]
```
Found hi-entropy localStorage:  Infinity  bits "https://www.youtube.com/watch?v=dQw4w9WgXcQ" yt-html5-player-modules::subtitlesModuleData::display-settings
```

It seems it's blocking first-party cookies, but I haven't noticed anything break yet.

**Comment by cooperq** [comment]
That message happens before it gets blocked, so it is not neccessarily indicative of youtube getting blocked. 
The infinity bits is a little concerning though (obviously a math error).

**Comment by rcutmore** [comment]
@cooperq I looked into this and the *estimateMaxEntropy* function in supercookie.js is responsible for this calculation. There's a comment stating the function may return Infinity when the given input is too long (currently located at [line 91](https://github.com/EFForg/privacybadgerfirefox/blob/master/data/supercookie.js#L91)).

On a side note, I was attempting to look into this but I couldn't get this function to execute. It's part of the script that gets returned by the *getScScript* function in that same file and it appears it is injected and then removed in at the end of the *insertScript* function. I assume the line `parent.insertBefore(script, parent.firstChild);` is what causes it to execute but I don't get any output from any console.log calls that I add.

**Comment by gunesacar** [comment]
I think this is expected behavior. We easily hit to the limits of the underlying floating point standard since the entropy estimation involves [exponentiation](https://github.com/EFForg/privacybadgerfirefox/blob/master/data/supercookie.js#L87).

For instance, a 250-character string with English lower-case letters [a-z] may contain `Math.pow(26, 250)` combinations.
```
>> console.log(Math.pow(26, 250))
Infinity
```
This shouldn't cause any problems though, since you can safely compare `Infinity` to other numbers, which is in our case `LOCALSTORAGE_ENTROPY_THRESHOLD` (33).
```
>> console.log(Infinity > 33)
true
```

**Comment by cooperq** [comment]
Seems like this isn't really a problem then. Commenting out that console.log message though to avoid confusion. Fixed in 67b788b5b8e590469930c652b5b426618fd70563
